datadog_checks_dev/tox.ini

```diff
@@ -8,7 +8,6 @@ envlist =
 
 [testenv]
 usedevelop = true
-skip_install = true
```

Comment:
Need this for https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/requirements-dev.txt